biopandas/mmcif/pandas_mmcif.py

```diff
@@ -479,3 +491,48 @@ def read_mmcif_from_list(self, mmcif_lines):
         # self.header, self.code = self._parse_header_code()
         self.code = self.data["entry"]["id"][0].lower()
         return self
+
+    def get_pandas_pdb(self, offset_chains: bool = True) -> PandasPdb:
+        """Returns a PandasPdb object with the same data as the PandasMmcif
+        object.
+
+        Attributes
+        ----------
+        offset_chains: bool
+            Whether or not to offset atom numbering based on number of chains.
+            This can arise due to the presence of TER records in PDBs which are
+            not found in mmCIFs.
+
+        """
+        pandaspdb = PandasPdb()
+        # for a in ["ATOM", "HETATM"]:
+        for a in self.df.keys():
```

Comment:
Hm, but doesn't that include the  `OTHERS` key as well? Maybe that could be more specific to the keys we actually care about

Reply:
Hmm, yes, good point. We could add an arg that defaults to `["ATOM", "HETATM"]` to allow some more control. What do you think?

Reply:
Yes, I think that's the "safer" choice